jnosql-mapping/jnosql-mapping-validation/src/main/java/org/eclipse/jnosql/mapping/validation/MappingValidator.java

```diff
@@ -84,16 +84,40 @@ public void validate(ConstructorEvent event) {
         }
     }
 
-    private Validator getValidator() {
-        if (!validators.isUnsatisfied()) {
-            return validators.get();
-        } else if (!validatorFactories.isUnsatisfied()) {
-            ValidatorFactory validatorFactory = validatorFactories.get();
-            return validatorFactory.getValidator();
-        } else {
-            ValidatorFactory factory = Validation.buildDefaultValidatorFactory();
-            return factory.getValidator();
-        }
+    /**
+     * Defines the {@link Validator} that it's going to be used during the validation process
+     *
+     * @throws NullPointerException when cannot to get the validator from CDI
+     */
+    @PostConstruct
+    public void postConstruct() {
```

Comment:
```suggestion
 void postConstruct() {
```
I guess there is no issue to don't defining it as public.

Reply:
@dearrudam you have the option to commit the suggestion; it only requires an accept on UI.

Reply:
@otaviojava I learned one more!!! Thanks!!!